src/now.js

```diff
@@ -435,6 +427,46 @@ const main = async (argv_) => {
     ctx.config.sh = Object.assign(ctx.config.sh, { user })
   }
 
+  if (typeof argv.team === 'string') {
+    const {team} = argv
+
+    if (team.length === 0) {
+      console.log(error(`You defined ${`--team`}, but it's missing a value`))
+      await exit(1)
+    }
+
+    if (isTTY) {
+      console.log(info('Caching team information...'))
+    }
+
+    const {token} = ctx.authConfig.credentials.find(item => item.provider === 'sh')
+
+    const headers = {
+      Authorization: `Bearer ${token}`
+    }
+
+    const url = `https://api.zeit.co/teams/?slug=${team}`
+    let body
+
+    try {
+      const res = await fetch(url, { headers })
+      body = await res.json()
+    } catch (err) {
+      console.error(error('Not able to load teams'))
+      await exit(1)
+    }
+
+    if (body.error) {
```

Comment:
Maybe let's check if it's e.g. `HTTP 403` and show `not authorized to access such team` or sth? 🙏